packages/web-components/fast-components/src/date-picker/date-picker.styles.ts

```diff
@@ -0,0 +1,170 @@
+import { css, ElementStyles } from "@microsoft/fast-element";
+import {
+    display,
+    ElementDefinitionContext,
+    FoundationElementDefinition,
+} from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+import { heightNumber } from "../styles/index";
+import {
+    accentForegroundActive,
+    bodyFont,
+    controlCornerRadius,
+    density,
+    designUnit,
+    disabledOpacity,
+    fillColor,
+    foregroundOnAccentActive,
+    neutralFillRest,
+    neutralForegroundRest,
+    strokeWidth,
+    typeRampBaseFontSize,
+    typeRampBaseLineHeight,
+    typeRampPlus3FontSize,
+    typeRampPlus3LineHeight,
+} from "../design-tokens";
+import { elevation } from "../styles/elevation";
+import { DirectionalStyleSheetBehavior } from "../styles";
+
+/**
+ * Styles for the fast-date-picker component
+ * @param context - control context
+ * @param definition - foundation element definition
+ * @returns styles
+ *
+ * @public
+ */
+export const DatePickerStyles: (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) => ElementStyles = (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) => css`
+    :host {
+        --elevation: 14;
+        --panel-size: calc(${designUnit} * (8 + ${density}) * 7px);
+        position: relative;
+    }
+
+    .flyout {
+        border: calc(${strokeWidth} * 1px) solid transparent;
+        border-radius: calc(${controlCornerRadius} * 1px);
+        padding: calc(${designUnit} * 5px);
+        background-color: ${fillColor};
+        ${elevation}
+        z-index: 1;
+        display: none;
```

Comment:
Use 'when' statements to show anchored region and not css.

Reply:
Updated.